jenkins/prepare-ps-build-docker.yml

```diff
@@ -52,16 +51,12 @@
                 stage('Build') {
                     steps {
                         parallel(
-                            "i386/centos:6":  { build('i386/centos:6') },
-                            "centos:6":       { build('centos:6') },
-                            "centos:7":       { build('centos:7') },
-                            "centos:8":       { build('centos:8') },
-                            "ubuntu:artful":  { build('ubuntu:artful') },
-                            "ubuntu:xenial":  { build('ubuntu:xenial') },
+                            "centos7":  { build('centos:7') },
+                            "centos8":  { build('centos:8') },
                             "ubuntu:bionic":  { build('ubuntu:bionic') },
```

Comment:
can we use numbers here?
like `ubuntu:18.04` instead of `ubuntu:bionic`
reason - it is simpler to compare with page - https://www.percona.com/services/policies/percona-software-support-lifecycle

Reply:
We can, but it will require to rework all pipelines in ps80 jenkins which use this docker